### ISSUE TITLE
Cannot find type definition file for 'core-js'.

### ISSUE THREAD

**Issue by aderbas** [swimlane/ng2d3#59]
I'm trying to use in my project (Angular 2 + angular-cli). I installed all dependencies by npm. If I install typings of core-js solve this issue but get many conflict errors like `Duplicate identifier 'PropertyKey' `. What it takes to make working properly?

(if you need any more information about my project, just ask)

**Comment by marjan-georgiev** [comment]
Could you share a github repo with a minimum version that reproduces this please?

**Comment by aderbas** [comment]
@marjan-georgiev 
https://github.com/aderbas/teste-app

**Comment by aderbas** [comment]
```
$ ng version
angular-cli: 1.0.0-beta.16
node: 6.8.1
os: linux x64
```

**Comment by aderbas** [comment]
If I install typings of core-js:

```
$ npm install @types/core-js --save-dev
$ ng build
 94% asset optimizationError in bail mode: [default] /var/www/html/teste-app/node_modules/@types/core-js/index.d.ts:21:13 
Duplicate identifier 'PropertyKey'.
aderbal@stewie:/var/www/html/teste-app$ npm install @types/core-js --save-dev
```

**Comment by amcdnl** [comment]
I actually fought this same issue for some time on another project and eventually just used a script to remove it and everything worked fine. 

@marjan-georgiev - maybe we should do same thing here? https://github.com/swimlane/angular2-data-table/blob/master/tasks/replace.js

**Comment by alanpurple** [comment]
1. install core-js typing by typings, not types by npm
2. add 
"files": [
    "typings/globals/core-js/index.d.ts"
  ]
to tsconfig.json

**Comment by alanpurple** [comment]
is core-js reference even necessary?

**Comment by aderbas** [comment]
I created project using angular-cli, the project created with all minimal dependencies necessary to work. Core-js is one of those dependencies.

**Comment by alanpurple** [comment]
@aderbas That's because angular project need core-js, not ng2d3

**Comment by amcdnl** [comment]
Yes u do need it.

**Comment by alanpurple** [comment]
@aderbas  even if it is necessary, angular has dependency on core-js, so ng2d3 compilation doesn't need core-js.

I remove all core-js reference line in ng2d3/release definition files, and it compiles and works fine

**Comment by aderbas** [comment]
@alanpurple you cloned the example I did to make this test?

**Comment by aderbas** [comment]
@alanpurple I did what you said and it really worked for my project test. But if I add 'NG2D3Module' in imports on file `app.module.ts` error appears in javascript bundle file, but chart work.

![ng2d3](https://cloud.githubusercontent.com/assets/12519019/19530888/184f4dde-960c-11e6-9f18-3774e3cc8c4a.png)

**Comment by alanpurple** [comment]
@aderbas  that's module system-related error and not related to ng2d3 I guess
I have much more complex code structure but doesn't have that error

**Comment by aderbas** [comment]
I understand. I questioned because when I remove `NG2D3Module` the error disappears. Thks

**Comment by amcdnl** [comment]
Should be fixed in 1.4.0
